### PR TITLE
Overview graphs

### PR DESCRIPTION
**Pull request by tpaternina**
Add the two overview graphs for the top 1 coin.
I parse the data from the API to streamline it to Charts.js 
Include a loading icon.

### CODE REVIEW

File: src/components/Chart/Chart.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Line } from "react-chartjs-2";
+
+export default function Chart(props) {
+  const yAxisMax = Math.max(...props.data);
+  const yAxisMin = Math.min(...props.data);
+
+  const data = {
+    labels: [...Array(props.data.length).keys()],
```
Comment (yoloOnTheBattlefield) [inline src/components/Chart/Chart.js:9]: can you explain this?

File: src/components/ChartOverview/ChartOverview.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    isLoading: false,
+    hasError: false,
+  };
+
+  getPrices = async () => {
+    try {
+      const { topCoin, currency } = this.props;
+      this.setState({ isLoading: true });
+      const query = `https://api.coingecko.com/api/v3/coins/${topCoin.id}/market_chart?vs_currency=${currency}&days=30&interval=daily`;
+      console.log(query);
```
Comment (yoloOnTheBattlefield) [inline src/components/ChartOverview/ChartOverview.js:30]: 😭

**Review by yoloOnTheBattlefield — APPROVED** [review]

File: src/components/Chart/Chart.js (hunk 1 above)
Comment (tpaternina) [reply src/components/Chart/Chart.js:9]: That's just to have an array with the integers from 0 to `data.length-1`

File: src/components/ChartVolumeOverview/ChartVolumeOverview.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  const { total_volumes } = props;
+  // Exclude 31st element which corresponds to ... average?
+  const data = {
+    datasets: [
+      {
+        data: total_volumes.map((item, index, array) => {
+          if (index !== array.length - 1) {
+            return formatOverviewChart(item);
+          }
+        }),
```
[the remark this answers is not in the document]
Comment (tpaternina) [reply src/components/ChartVolumeOverview/ChartVolumeOverview.js:17]: True. But I'm also applying a function (`formatOverviewChart()`) on each item. Should I chain .filter().map() ?

[the remark this answers is not in the document]
Comment (yoloOnTheBattlefield) [reply src/components/ChartVolumeOverview/ChartVolumeOverview.js:17]: yes please!

**Pull request merged**